engine/src/main/java/org/entando/entando/aps/system/services/api/model/ApiError.java

```diff
@@ -15,55 +15,56 @@
 
 import java.io.Serializable;
 
-import javax.ws.rs.core.Response;
-import javax.ws.rs.core.Response.Status;
 import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlType;
+import org.springframework.http.HttpStatus;
 
 /**
  * @author E.Santoboni
  */
 @XmlRootElement(name = "error")
 @XmlType(propOrder = {"code", "message"})
 public class ApiError implements Serializable {
-	
-	public ApiError() {}
-	
-	public ApiError(String code, String message) {
-		this.setCode(code);
-		this.setMessage(message);
-	}
-	
-	public ApiError(String code, String message, Response.Status status) {
-		this.setCode(code);
-		this.setStatus(status);
-		this.setMessage(message);
-	}
-	
-	public String getCode() {
-		return _code;
-	}
-	public void setCode(String code) {
-		this._code = code;
-	}
-	
-	public String getMessage() {
-		return _message;
-	}
-	public void setMessage(String message) {
-		this._message = message;
-	}
-	
-	public Status getStatus() {
-		return _status;
-	}
-	protected void setStatus(Status status) {
-		this._status = status;
-	}
-	
-	private String _code;
-	private String _message;
-	
-	private Response.Status _status;
-	
+
+    private String code;
+    private String message;
+    private HttpStatus status;
+
+    public ApiError() {
```

Comment:
i fear that someone in the new API could use this so i would like change name in something like LegacyApiError if possible but it's just a suggestion ...

Reply:
Renamed to `LegacyApiError`.